### PR TITLE
Update `wrangler types` documentation with new customization options

### PR DESCRIPTION
**Pull request by dario-piotrowicz**
update the `wrangler types` documentation to add the new `path` positional parameter and `--env-interface` flag

### CODE REVIEW

File: content/workers/wrangler/commands.md [hunk 1]
```diff
@@ -2326,8 +2326,21 @@ Deleted certificate 99f5fef1-6cc1-46b8-bd79-44a0d5082b8d successfully
...
-```txt
-wrangler types
+```sh
+wrangler types [<PATH>] [OPTIONS]
 ```
 
+{{<definitions>}}
+
+- `PATH` {{<type>}}string{{</type>}} {{<prop-meta>}}(default: `worker-configuration.d.ts`){{</prop-meta>}}
+  - The path to where to write the resulting types n entry point for your Worker.
```
Comment (petebacondarwin) [inline content/workers/wrangler/commands.md]: "n"?

Comment (petebacondarwin) [reply content/workers/wrangler/commands.md]: Perhaps:

```suggestion
  - The path to where the declaration file for your Worker will be written.
```

File: content/workers/wrangler/commands.md [hunk 2]
```diff
@@ -2326,8 +2326,21 @@ Deleted certificate 99f5fef1-6cc1-46b8-bd79-44a0d5082b8d successfully
...
+  - Since it is a declaration file, it needs to have a `d.ts` extension.
+
+- `--env-interface` {{<type>}}string{{</type>}} {{<prop-meta>}}(default: `Env`){{</prop-meta>}}
+  - The name of the interface to generate for the environment object.
+  - _Ignored if the worker uses the Service Worker syntax._
```
Comment (petebacondarwin) [inline content/workers/wrangler/commands.md]: Not valid if the worker uses Service Worker syntax.

**Review by petebacondarwin — CHANGES_REQUESTED** [review]

File: content/workers/wrangler/commands.md (hunk 2 above)
Comment (dario-piotrowicz) [reply content/workers/wrangler/commands.md]: 🤦 yeah I need to update this thanks! 🙇

File: content/workers/wrangler/commands.md (hunk 1 above)
Comment (dario-piotrowicz) [reply content/workers/wrangler/commands.md]: > "n"?

sorry... I have no idea what happened there 😓

File: content/workers/wrangler/commands.md (hunk 2 above)
Comment (dario-piotrowicz) [reply content/workers/wrangler/commands.md]: Updated 🙂👍

**Review by petebacondarwin — APPROVED** [review]
Great!

File: content/workers/wrangler/commands.md [hunk 3]
```diff
@@ -2326,8 +2326,21 @@ Deleted certificate 99f5fef1-6cc1-46b8-bd79-44a0d5082b8d successfully
...
-wrangler types
+```sh
+wrangler types [<PATH>] [OPTIONS]
 ```
 
+{{<definitions>}}
+
+- `PATH` {{<type>}}string{{</type>}} {{<prop-meta>}}(default: `worker-configuration.d.ts`){{</prop-meta>}}
+  - The path to where the declaration file for your Worker will be written.
+  - Since it is a declaration file, it needs to have a `d.ts` extension.
```
Comment (deadlypants1973) [inline content/workers/wrangler/commands.md]:
```suggestion
  - A declaration file needs to have a `d.ts` extension.
```

File: content/workers/wrangler/commands.md [hunk 4]
```diff
@@ -2326,8 +2326,21 @@ Deleted certificate 99f5fef1-6cc1-46b8-bd79-44a0d5082b8d successfully
...
+
+- `--env-interface` {{<type>}}string{{</type>}} {{<prop-meta>}}(default: `Env`){{</prop-meta>}}
+  - The name of the interface to generate for the environment object.
+  - _Not valid if the worker uses the Service Worker syntax._
```
Comment (deadlypants1973) [inline content/workers/wrangler/commands.md]:
```suggestion
  - Not valid if the Worker uses the Service Worker syntax.
```

**Review by deadlypants1973 — APPROVED** [review]

File: content/workers/wrangler/commands.md (hunk 3 above)
Comment (petebacondarwin) [reply content/workers/wrangler/commands.md]: I don't think that suggestion quite captures the validation here. What we are trying to say is that this option is invalid and will error if it does not end in `.d.ts`.

Perhaps:

```suggestion
  - The path to the declaration file must have a `d.ts` extension.
```

Comment (dario-piotrowicz) [reply content/workers/wrangler/commands.md]: Thanks @petebacondarwin that looks good to me 😄👍 

Hopefully @deadlypants1973 agrees 🙂

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**